v3.1/how-to/maintain/backup-and-restore/br-best-practice.md

```diff
@@ -0,0 +1,403 @@
+
+### 准备工作
+
+BR backup 命令的详细使用方法请参考[文档](/v3.1/how-to/maintain/backup-and-restore/br.md#BR命令行描述)。
+
+#### 备份准备工作
+
+1. 运行 BR backup 命令前，查询 TiDB 集群的 GC 值并使用 MySQL 客户端将其调整为合适的值，确保备份期间不会发生 GC。
+
+    ```
+    SELECT * FROM mysql.tidb WHERE VARIABLE_NAME = 'tikv_gc_life_time';
+    UPDATE mysql.tidb SET VARIABLE_VALUE = '720h' WHERE VARIABLE_NAME = 'tikv_gc_life_time';
+    ```
+
+2. 在备份完成后，将该参数调回原来的值。
+
+    ```
+    update mysql.tidb set VARIABLE_VALUE = '10m' where VARIABLE_NAME = 'tikv_gc_life_time';
+    ```
+
+BR restore 命令的详细使用方法请参考[文档](/v3.1/how-to/maintain/backup-and-restore/br.md#BR命令行描述)。
+
+#### 恢复准备工作
+
+1. 运行 BR restore 前检查新集群没有同名 table。
+2. （可选， only recommended for ≤3.1.0-beta）运行 BR restore 前关闭 pd leader schedulers 提升恢复性能。
+
+   ```
+   ./pd-ctl -u 172.16.5.198:2379 scheduler remove balance-leader-scheduler
+   ./pd-ctl -u 172.16.5.198:2379 scheduler remove balance-hot-region-scheduler
+   ./pd-ctl -u 172.16.5.198:2379 scheduler remove balance-region-scheduler
+   ```
+
+   同时在恢复数据后，要将删除的 scheduler 添加回来
+
+   ```
+   ./pd-ctl -u 172.16.5.198:2379 scheduler add balance-leader-scheduler
+   ./pd-ctl -u 172.16.5.198:2379 scheduler add balance-hot-region-scheduler
+   ./pd-ctl -u 172.16.5.198:2379 scheduler add balance-region-scheduler
+   ```
+
+### 单表数据备份到网络盘（推荐）
+
+通过 BR backup 命令，将单表数据 --db batchmark --table order_line 备份到指定的网络盘路径下 local:///br_data
+
+#### 前置要求
+
+* 配置一台高性能 SSD 硬盘主机作为 nfs server 存储数据，其他所有 BR 节点和 TiKV 节点作为 nfs client 挂载相同路径(如 /br_data)到 nfs server 上。
+
+* nfs server 和 nfs client 的传输速率至少要达到备份集群的 TiKV 实例数 * 150MB/s。否则网络 IO 有可能成为性能瓶颈。
+
+#### 部署拓扑
+
+![img](/media/br/backup-nfs-deploy.png)
+
+#### 运行备份
+
+备份前在 TiDB 里通过 `admin checksum table order_line` 获得备份的目标表 --db batchmark --table order_line 统计信息如下
+
+![img](/media/br/total-data.png)
+
+备份前，调 GC 可以参考[备份准备工作](#备份准备工作)
+
+运行 BR backup 命令
+
+```
+bin/br backup table --db batchmark --table order_line -s local:///br_data --pd 172.16.4.217:2379 --log-file backup-nfs.log --concurrency 16
+```
+
+#### <span id="backup-status">运行指标</span>
+
+Backup CPU Utilization  - 参与备份的 TiKV 节点（backup-worker）和 br 节点（backup-endpoint）CPU 使用率）
+
+![img](/media/br/backup-cpu.png)
+
+IO Utilization - 参与备份的 TiKV 节点的 IO 使用率
+
+![img](/media/br/backup-io.png)
+
+BackupSST Generation Throughput - 参与备份的 TiKV 节点生成 backupSST 文件的吞吐（正常时单台 TiKV 吞吐在 150MB/s 左右）
+
+![img](/media/br/backup-throughput.png)
+
+One Backup Range Duration - 备份一个 range 操作耗时，分为 scan 耗时（scan KV）和 save 耗时（保存成 backupSST）
+
+ ![img](/media/br/backup-range-duration.png)
+
+One Backup Subtask Duration - 一次备份会拆分成多个子任务运行，监控显示了子任务的耗时，虽然是备份单表，但因为表中有3个索引，所以正常会划分成4个子任务，图中有13个点，说明有9次（13-4）重试，备份过程可能产生 region 调度行为，少量重试是正常的。
+
+![img](/media/br/backup-subtask-duration.png)
+
+Backup Errors - 备份过程中的错误（正常时无错误，如果出现少量错误也不要担心，Backup 有重试机制，可能会导致备份时间增加，但不会影响正确性）
+
+![img](/media/br/backup-errors.png)
+
+Checksum Request duration - 对备份集群执行 admin checksum 的耗时统计
+
+![img](/media/br/checksum-duration.png)
+
+#### 结果解读
```

Comment:
This section looks more like a list of results that lacks descriptions. Are these the "best results" users might get from the above `best practices`? Could you make the purpose clearer?

Reply:
okay, I add some description, it's hard to say these are "best results", it's just translate related statistics from log file, make user understand these results